jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/ToolkitAuthManager.kt

```diff
@@ -155,7 +161,7 @@ fun loginSso(project: Project?, startUrl: String, region: String = DEFAULT_SSO_R
             ManagedSsoProfile(
                 region,
                 startUrl,
-                scopes
+                allScopes.toSet().toList()
```

Comment:
the dataclass itself should be a set but let's handle that in a different pr or backlog it